_docs-2/configuration/server-properties.md

```diff
@@ -119,6 +119,7 @@ Below are properties set in `accumulo.properties` or the Accumulo shell that con
 | <a name="monitor_prefix" class="prop"></a> **monitor.*** | **Available since:** 1.3.5<br>Properties in this category affect the behavior of the monitor web server. |
 | <a name="monitor_lock_check_interval" class="prop"></a> monitor.lock.check.interval | **Available since:** 1.5.1<br>The amount of time to sleep between checking for the Monitor ZooKeeper lock<br>**type:** TIMEDURATION, **zk mutable:** no, **default value:** `5s` |
 | <a name="monitor_port_client" class="prop"></a> monitor.port.client | **Available since:** 1.3.5<br>The listening port for the monitor's http service<br>**type:** PORT, **zk mutable:** no, **default value:** `9995` |
+| <a name="monitor_port_log4J" class="prop"></a> monitor.port.log4j | **Available since:** 1.3.5<br>The listening port for the monitor's http service<br>**type:** PORT, **zk mutable:** no, **default value:** `9995` |
```

Comment:
```suggestion
| <a name="monitor_port_log4j" class="prop"></a> monitor.port.log4j | **Available since:** 1.3.5<br>The listening port for the monitor's http service<br>**type:** PORT, **zk mutable:** no, **default value:** `9995` |
```
It looks like this is case sensitive. The link in the network table on the `In-depth Installation` page should work with this change.